cmd/enumerate_github/README.md

```diff
@@ -0,0 +1,130 @@
+# GitHub Enumeration Tool
+
+This tool is used to reliably enumerate projects on GitHub.
+
+The output of this tool is used as an input for the `collect_signals` tool. 
+
+## Example
+
+```shell
+$ export GITHUB_TOKEN=ghp_x  # Personal Access Token Goes Here 
+$ enumerate_github \
+    -start 2008-01-01 \
+    -min-stars=10 \
+    -workers=1 \
+    github_projects.txt
+```
+
+## Install
+
+```shell
+$ go install github.com/ossf/criticality_score/cmd/enumerate_github
+```
+
+## Usage
+
+```shell
+$ enumerate_github [FLAGS]... FILE
+```
+
+The URL for each repository is written to `FILE`. If `FILE` is `-` the results will be written to STDOUT.
+
+### Flags
+
+#### Output flags
+
+- `-append` appends output to `FILE` if it already exists.
+- `-force` overwrites `FILE` if it already exists and `-append` is not set.
+
+If `FILE` exists and neither `-append` nor `-force` is set the command will fail.
+
+#### Date flags
+
+- `-start date`
+        the start date to enumerate back to. Must be at or after `2008-01-01`. Defaults to `2008-01-01`.
+- `-end date`
+        the end date to enumerate from. Defaults to today's date.
+
+#### Query/Star flags
+
+- `-min-stars int` only enumerates repositories with this or more of stars
+  Defaults to `10`.
+- `-query string` sets the base query to use for enumeration. Defaults to
+  `is:public`. See GitHub's [search help](https://docs.github.com/en/search-github/searching-on-github/searching-for-repositories)
+  for more detail.
+- `-require-min-stars` abort execution if `-min-stars` can't be reached during
+  enumeration. If not set some repositories created on a certain date may not
+  be included.
+- `-star-overlap int` the number of stars to overlap between queries. Defaults
+  to `5`. A an overlap is used to avoid missing repositories whose star count
+  changes during enumeration.
+
+#### Misc Flags
+
+- `-log level` set the level of logging. Can be `debug`, `info` (default), `warn` or `error`.
+- `-workers int` the total number of concurrent workers to use. Default is `1`.
+
+### Authentication
+
+Expects an environment variable with one or more GitHub Personal Access Tokens,
```

Comment:
Maybe more clearly worded as "A comma delimited environment variable with one or more GitHub Personal Access Tokens must be set" ?

Reply:
Done. I also moved this section above the Flags documentation.